1-exercises/E-while-loop-with-array/exercise.js

```diff
@@ -5,19 +5,30 @@
 */
 
 const BIRTHDAYS = [
-    "January 7th",
-    "February 12th",
-    "April 3rd",
-    "April 5th",
-    "May 3rd",
-    "July 11th",
-    "July 17th",
-    "September 28th",
-    "November 15th"
+  "January 7th",
+  "February 12th",
+  "April 3rd",
+  "April 5th",
+  "May 3rd",
+  "July 11th",
+  "July 17th",
+  "September 28th",
+  "November 15th",
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  // TODO
+  //   for (let i = 0; i < BIRTHDAYS.length; i++) {
```

Comment:
Remember to remove old comments. Remember you still have old versions of your code in your version history - so you should be able to inspect the old version if you want to.